### PR TITLE
Languages new

### PR DESCRIPTION
**Pull request by smolamarcin**

### CODE REVIEW

File: battleship-client/src/main/java/com/academy/solid/nie/client/ui/StartSceneController.java [hunk 1]
```diff
@@ -4,28 +4,39 @@
...
- *
+ * Controller to manage components of the starting window.
  */
 public final class StartSceneController {
     @FXML
     private ImageView logo;
     @FXML
     private Button btnConnect;
+    @FXML
+    private Button polish;
```
Comment (rivalez) [inline battleship-client/src/main/java/com/academy/solid/nie/client/ui/StartSceneController.java]: rename to polishBtn

File: battleship-client/src/main/java/com/academy/solid/nie/client/ui/StartSceneController.java [hunk 2]
```diff
@@ -4,28 +4,39 @@
...
+    @FXML
+    private Button english;
```
Comment (rivalez) [inline battleship-client/src/main/java/com/academy/solid/nie/client/ui/StartSceneController.java]: rename to englishBtn

Comment (rivalez) [reply battleship-client/src/main/java/com/academy/solid/nie/client/ui/StartSceneController.java]: or even englishLangBtn

File: battleship-client/src/main/resources/POLISH.properties [hunk 3]
```diff
@@ -0,0 +1,4 @@
+WELCOME=Witaj w  mojej grze.
+WRONG_IP=Nieprawidlowe IP!
```
Comment (rivalez) [inline battleship-client/src/main/resources/POLISH.properties]: uzyj polskich znakow asci

File: battleship-client/src/main/resources/POLISH.properties [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+WIN= Wygrales!
+LOSE= Przegrales!
```
Comment (Ozymadias) [inline battleship-client/src/main/resources/POLISH.properties]: can't we use here polish letters?

**Pull request merged**